README.md

```diff
@@ -16,3 +16,25 @@ As a user, I would like to configure my feature to use a single global content s
 
 ## What's next?
 Now that we have a feature that renders based on the global content instead of being fixed to component level content, we want to see how we can dynamically route content using PageBuilder templates. Switch to branch `fusion-12` to learn more about this next step.
+
+## New Task: User Story 12
+As a user, I would like to be able to render many pieces of content via this template in the browser.
+
+## GUIDE
+No code changes are required to complete this task
+
+1. Create a new template in PB called `Articles`
+
+2. Using the basic layout, add in the header feature and any other feature you'd like
+
+3. Under "Developer Tools" > "Resolvers", create a new resolver
+
+4. Fill in the details of your resolver as follows:
+- Resolver Name: give your resolver a name
+- Pattern: set a URL pattern for this resolver to target (for our demo example, you can simply put `/` for all urls)
+- Priority: set the priority for your resolver relative to other resolvers you might have (you can set it to `1`)
+- Default Template: set the default template to render for all url patterns that matches what is definied for this resolver. In our case, this wil be our `Articles` template
+- Content Source: set the content source the resolver should use to fetch the articles content. For our example, it will be `content-api`
+- website_url & published: set the website url and published value - you can use any value here for our example and save our resolver
```

Comment:
Do you think it would be helpful to discuss the difference between static and pattern values and how to retrieve the pattern values from the resolver pattern or is that too much of a deep dive?

Reply:
actually yes - I think having a high level description of the two would be helpful here